Automattic/wp-calypso
# Stats Chart: Add comparisons for chart tabs

Related to https://github.com/Automattic/red-team/issues/268

## Proposed Changes

* Add comparison for stats tabs

## Testing Instructions



* Open Calypso Live `/stats/day/jetpack.com?flags=stats%2Fnew-date-filtering`
* Ensure the tabs below chart look okay in different breakpoints
* Ensure the comparison data is correct - current set to compare with the previous exact same range from selected

<img width="1307" alt="image" src="https://github.com/user-attachments/assets/bfc305d5-4f64-4af4-8fec-1a6f892affc5">

Kudos to dognose for adding all the styles. Very much appreciated 🙇 

## Pre-merge Checklist
